ui/cypress/e2e/tasks.test.ts

```diff
@@ -24,25 +24,6 @@ describe('Tasks', () => {
     })
   })
 
-  it('cannot create a task with an invalid to() function', () => {
```

Comment:
This test was also deleted from Cloud (I assume for the same reasons), see influxdata/ui@ca989db68d56a4ab36f871a7745c1608c48f0b6b